app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -84,5 +88,41 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.$module.cookieBannerConfirmationMessage.style.display = 'block'
   }
 
+  CookieBanner.prototype.listenForCrossOriginMessages = function () {
+    window.addEventListener('message', this.receiveMessage.bind(this), false)
+  }
+
+  CookieBanner.prototype.receiveMessage = function (event) {
+    var trustedDomain = 'publishing.service.gov.uk'
+    var origin = event.origin
+
+    // Return if no origin is given or the browser doesn't support lastIndexOf
+    if (!origin || !origin.lastIndexOf) {
+      return
+    }
+
+    // Polyfill origin.endsWith(trustedDomain) for IE
+    var offset = trustedDomain && this ? origin.length - trustedDomain.length : -1
```

Comment:
Pretty sure we don't need to test `trustedDomain` exists here.

Reply:
Good spot; leftover from refactoring